### PR TITLE
Update enable epochs handler

### PR DESCRIPTION
**Pull request by sstanculeanu**

### CODE REVIEW

File: vmhost/hostCore/flags.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package hostCore
+
+import "github.com/multiversx/mx-chain-core-go/core"
+
+const (
```
Comment (iulianpascalau) [inline vmhost/hostCore/flags.go:5]: add a comment here valid for all consts?

Comment (sstanculeanu) [reply vmhost/hostCore/flags.go:5]: pushed

**Review by iulianpascalau — APPROVED** [review]

**Review by iulianpascalau — DISMISSED** [review]

File: testcommon/testInitializer_inputs.go [hunk 2]
```diff
@@ -363,28 +362,12 @@ func DefaultTestVMWithGasSchedule(
...
-			IsGlobalMintBurnFlagEnabledField:                     true,
-			IsCheckFunctionArgumentFlagEnabledField:              true,
-			IsCheckExecuteOnReadOnlyFlagEnabledField:             true,
-		},
+		VMType:                   DefaultVMType,
+		BlockGasLimit:            uint64(1000),
+		GasSchedule:              gasSchedule,
+		BuiltInFuncContainer:     builtInFunctions.NewBuiltInFunctionContainer(),
+		ProtectedKeyPrefix:       []byte("E" + "L" + "R" + "O" + "N" + "D"),
+		EnableEpochsHandler:      &mock.EnableEpochsHandlerStub{},
```
Comment (bogdan-rosianu) [inline testcommon/testInitializer_inputs.go:370]: previously more flags were set to true. do we need to do this now as well?

Comment (sstanculeanu) [reply testcommon/testInitializer_inputs.go:370]: they were not needed

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**